src/components/page-deployment/deployment-bread-crumb.js

```diff
@@ -1,21 +1,24 @@
-import Breadcrumb from '../breadcrumb';
 import { routeWithParams, smallDeploymentName } from '../../utils/string';
 import routes from '../../routes';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Breadcrumbs } from '@equinor/eds-core-react';
 
 const DeploymentBreadCrumb = ({ appName, deploymentName }) => {
   return (
-    <Breadcrumb
-      links={[
-        { label: appName, to: routeWithParams(routes.app, { appName }) },
-        {
-          label: 'Deployments',
-          to: routeWithParams(routes.appDeployments, { appName }),
-        },
-        { label: smallDeploymentName(deploymentName) },
-      ]}
-    />
+    <Breadcrumbs>
+      <Breadcrumbs.Breadcrumb href={routeWithParams(routes.app, { appName })}>
```

Comment:
react-router does not work with anchor elements rendered by Breeadcrumb. Remove href from Breadcrumb and add a NavLink component (see usage in the internal web-console breadcrumb component).